openslides/motions/static/js/motions/pdf.js

```diff
@@ -853,86 +854,113 @@ angular.module('OpenSlidesApp.motions.pdf', ['OpenSlidesApp.core.pdf'])
             };
 
             var createTOContent = function() {
-                var heading = {
+                var toc = [];
+                var exportCategory = tocWithCategory();
```

Comment:
You can just write `var exportCategory = (sorting === 'identifier' || sorting === 'category.prefix')`